sqlmesh/core/state_sync/engine_adapter.py

```diff
@@ -657,6 +657,7 @@ def remove_interval(
                             exp.select("name", "identifier", "version")
                             .from_(self.intervals_table)
                             .where(where)
+                            .distinct()
```

Comment:
why were there ever duplicates?

Reply:
Wdym? There can be multiple interval records per snapshot which is completely normal and expected

Reply:
ah i see, this is just fetching if any interval exists, you don't actually care about the interval